modules/installation-configuration-parameters.adoc

```diff
@@ -701,11 +701,15 @@ link:https://yaml.org/spec/1.2/spec.html#sequence//[YAML sequence].
 
 |`compute.aws.region`
 |The AWS region that the installation program creates compute resources in.
-|Any valid link:https://docs.aws.amazon.com/general/latest/gr/rande.html[AWS region], such as `us-east-1`.
+|Any valid link:https://docs.aws.amazon.com/general/latest/gr/rande.html[AWS region], such as `us-east-1`. You can use the AWS CLI to access the regions available based on your selected instance type. For example:
```

Comment:
@Prashanth684 I added the cli command, but I'm not sure if I like the formating/wording of how it is right now. But I was wondering if the command itself looked correct?

Reply:
yes, the command is correct - this is the sample output:

```
{
    "InstanceTypeOfferings": [
        {
            "InstanceType": "c7g.xlarge",
            "LocationType": "region",
            "Location": "us-east-2"
        }
    ]
}

```

i don't think it is necessary to add the output here and also i have an older version of the cli, so it shows only one region.

Reply:
Gotcha! Thank you!